daml-foundations/daml-ghc/tests/daml-ghc-deterministic.sh

```diff
@@ -2,6 +2,32 @@
 # All rights reserved.
 set -euo pipefail
 
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
```

Comment:
Should we move this to some shared file that we include instead of copying it everywhere?

Reply:
@cocreature Unfortunatelly there's no other way to do it in `sh_test` as there's a chicken-and-egg problem there - to source the `rlocation` bash function you need to perform pseudo rlocation - see this comment: https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash#L34

Reply:
Ugh, that is annoying. Thanks for the explanation!